acceptance/tests/ticket_2455_on_solaris_init_provider_should_start_service_in_own_smf_contract.rb

```diff
@@ -0,0 +1,98 @@
+test_name "(PUP-2455) Service provider should start Solaris init service in its own SMF contract"
+
+step "Setup fixture service manifest on master"
+
+  testdir = master.tmpdir('solaris_services_in_own_smf_contract')
+
+  test_manifest = <<MANIFEST
+File {
+  ensure => directory,
+  mode => "0750",
+  owner => #{master.puppet['user']},
+  group => #{master.puppet['group']},
+}
+file { '#{testdir}': }
+file { '#{testdir}/environments': }
+file { '#{testdir}/environments/production': }
+file { '#{testdir}/environments/production/manifests': }
+file { '#{testdir}/environments/production/manifests/site.pp':
+  ensure  => file,
+  content => '
+    node default {
+      service { "vmware-tools":
```

Comment:
Could we just have a fixture that runs a dummy service? If puppet is there we know we have ruby. A dumb daemon is only a couple lines of code...